docs/ops/comparison/Greater_1.md

```diff
@@ -4,7 +4,16 @@
 
 **Category**: Comparison binary operation
 
-**Short description**: *Greater* performs element-wise comparison operation with two given tensors applying multi-directional broadcast rules.
+**Short description**: *Greater* performs element-wise comparison operation with two given tensors applying broadcast rules specified in the `auto_broadcast` attribute.
+
+**Detailed description**
+Before performing arithmetic operation, input tensors *a* and *b* are broadcasted if their shapes are different and `auto_broadcast` attributes is not `none`. Broadcasting is performed according to `auto_broadcast` value.
```

Comment:
`auto_broadcast` attributes is not ---> either `auto_broadcast` attribute is or `auto_broadcast` attributes are